### ISSUE TITLE
re-installing rmapi fail

### ISSUE THREAD

**Issue by mgs49** [juruen/rmapi#168]
Hi - big fan of rmapi and allowing me to print to the reM 2 from any application on MacOS is just brilliant.
However I had to deinstall some things (including the reM connections) and then wanted to re-install the print-to-reM function and that failed:

michael@Michaels-MacBook ~ % ./rmapi
ERROR: 2021/04/07 12:13:48 auth.go:99: failed to create a new user token

I did not get a prompt that asks for the new reM token from my.remarkable.com

Any help very welcome !

**Comment by ddvk** [comment]
do you have the latest version?

**Comment by ddvk** [comment]
otherwise remove $HOME/Library/Application Support/rmapi/rmapi.config

**Comment by mgs49** [comment]
Thanks, but unfortunately the rmapi directory does not exist (even when I enable seeing hidden files)
What now ?

**Comment by ddvk** [comment]
do you have a `~/.rmapi.conf` if yes remove that

**Comment by mgs49** [comment]
I’m afriad I have nothing with rmapi (no directory, no files, …) anywhere. Only the rmapi executable file in the user directory.
When I invoke that I get

Last login: Tue Apr 20 18:54:37 on ttys000
michael@Michaels-MBP ~ % ./rmapi
ERROR: 2021/04/20 18:55:28 auth.go:99: failed to create a new user token
michael@Michaels-MBP ~ %

**Comment by ddvk** [comment]
so then the config it is defintely somewhere in ~/Library/Application Support/rmapi/

otherwise, you can try installing the latest master which should fix it

**Comment by mgs49** [comment]
Yes, I do have the latest version. In fact, I again deleted rmapi and then re-loaded it just to make sure (version 0.0.13). 
Same again:

Last login: Tue Apr 20 19:43:41 on ttys000
  [Bookmarked 23. Apr 2021 at 12:22:44]
  [Bookmarked 23. Apr 2021 at 12:22:44]
michael@Michaels-MBP ~ % ./
zsh: permission denied: ./
michael@Michaels-MBP ~ % curl -L  https://github.com/juruen/rmapi/releases/download/v0.0.13/rmapi-macosx.zip -o rmapi.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   624  100   624    0     0    968      0 --:--:-- --:--:-- --:--:--   967
100 9690k  100 9690k    0     0  2663k      0  0:00:03  0:00:03 --:--:-- 3705k
michael@Michaels-MBP ~ % unzip rmapi.zip
Archive:  rmapi.zip
  inflating: rmapi                   
michael@Michaels-MBP ~ % ./rmapi
ERROR: 2021/04/23 12:24:09 auth.go:99: failed to create a new user token
michael@Michaels-MBP ~ %

**Comment by ddvk** [comment]
i meant the latest development version from `master`

**Comment by mgs49** [comment]
I am only a simple user, but I asked a friend to recompile it and construct a new rmapi exceutable.
Then, however I get the error that the developer is not authorised:

Could you make a new rmapi with the latest and put it available for download ?

Thanks and regards
Michael

**Comment by juruen** [comment]
> Could you make a new rmapi with the latest and put it available for download ? Thanks and regards Michael

I'm on it!

**Comment by juruen** [comment]
@mgs49 the new `0.0.14` release is now available to download.

Thanks!

**Comment by mgs49** [comment]
The latest version (0.0.14) works ! Many thanks !
rmapi is soooo useful …

I don’t want to stretch your patience but  mget/mput sound super useful too. Are they a way of doing a local backup of the reM cloud ? If so can you say how to use it ?

Many thanks
Michael

**Comment by ddvk** [comment]
just do `rmapi mget -i -o somefolder` on a regular basis. that should get only new and modified documents. you can then restore a document by doing `rmapi put document.zip`
